tools/Makefile

```diff
@@ -84,19 +46,39 @@ VPATH := $(filter-out -I,$(INCLUDES))
 .PHONY: all allopt opt.opt # allopt and opt.opt are synonyms
 allopt: opt.opt
```

Comment:
It's presently "wrong", but it would make sense these two lines down the `all:` and `opt.opt` lines below and make `all` the default target (at presently, confusingly, `allopt` is the default target)

Reply:
I think you accidentally a verb?

Reply:
Ah, "to move", etc.. I regularly from verb-disappearance and words order a funny in.

Reply:
The key point was to make `all` the first target 🙂

Reply:
Done, explanation for the thanks.